cardano-launcher/src/Cardano/Shell/Launcher.hs

```diff
@@ -106,12 +122,12 @@ runWallet ed walletPath walletArguments updaterData = do
     -- Let us map the interesting commands into a very simple "language".
     let exitCode :: DaedalusExitCodes
         exitCode = case walletExitStatus of
-            ExitFailure 20  -> RunUpdate
-            ExitFailure 21  -> RestartInGPUSafeMode
-            ExitFailure 22  -> RestartInGPUNormalMode
+            ExitFailure 20 -> RunUpdate
```

Comment:
Please, stop reformatting the code that you don't need unless you have a specific improvement.